api/api/spec/spec.yaml

```diff
@@ -4133,6 +4133,8 @@ components:
             $ref: '#/components/schemas/LogSummary'
           wazuh-modulesd:syscollector:
             $ref: '#/components/schemas/LogSummary'
+          wazuh-modulesd:vulnerability-detection:
```

Comment:
```suggestion
          wazuh-modulesd:vulnerability-scanner:
```
Checked the following issue for references regarding the VD logs: https://github.com/wazuh/wazuh/issues/21789#issuecomment-1932540280 and also launched a manager with the PR's branch:
```
2024/02/08 10:37:33 wazuh-modulesd:vulnerability-scanner: INFO: Starting vulnerability_scanner module.
```